docs/src/martin-cp.md

```diff
@@ -1,18 +1,20 @@
 # Generating Tiles in Bulk
 
-`martin-cp` is a tool for generating tiles in bulk, from any source(s) supported by Martin, and save retrieved tiles
-into a new or an existing MBTiles file. `martin-cp` can be used to generate tiles for a large area or multiple areas
+We offer the `martin-cp` tool for generating tiles in bulk, from any source(s) supported by Martin, and save retrieved tiles
+into a new or an existing MBTiles file.
```

Comment:
I'm wondering why breaks this to a new line? Is there some linting rule for this? For readability?

Reply:
This is already in this file for quite some time. Likely some editor config back in the day or someone really liking 80 cols and not having their editor configured like I have:

![image](https://github.com/user-attachments/assets/5d79f214-5667-4e18-8463-b804677e8ea1)

Reply:
Guessing breaking to lines is good to git diff?  I'm ok with this. Just wondering.